whois-client/src/main/java/net/ripe/db/whois/api/rest/domain/WhoisObject.java

```diff
@@ -59,6 +63,10 @@ public class WhoisObject {
     @XmlAttribute
     private Integer version;
 
+    @XmlElement(name = "objectInfoMessages")
+    @XmlJavaTypeAdapter(value = ValidListXmlAdapter.class)
```

Comment:
Is @XmlElementWrapper can do the trick ? instead of creating an adaptper ?

Reply:
I didn't try, for now I will remove the adapter when applying Eds solution.
I take note about that Wrapper

Reply:
Removed